src/main/java/org/jitsi/videobridge/Content.java

```diff
@@ -255,7 +255,8 @@ void askForKeyframes(Collection<Endpoint> endpoints)
     public RtpChannel createRtpChannel(String channelBundleId,
                                        String transportNamespace,
                                        Boolean initiator,
-                                       RTPLevelRelayType rtpLevelRelayType)
+                                       RTPLevelRelayType rtpLevelRelayType,
+                                       Integer receiveSimulcastLayer)
```

Comment:
Hi @lluczo, thank you very much for this! :+1:

One minor note, I would like to avoid modifying the create `createRtpChannel` method. Could you please create a setter for the `receiveSimulcastLayer` parameter in the `VideoChannel` and use that in Videobridge.java?\

Thank you very much :)